### DISCUSSION TITLE
how to show special files e.g. .htaccess?

### DISCUSSION THREAD

**Discussion by hjstrauss** [prasathmani/tinyfilemanager#764] [Q&A] [answered]
Hi,
is there any configuration setting to activate the display of files starting with a '.'? I installed tinyfilemanager today and am not able to see files like for example .htaccess .user.ini and similar files. It seems as if files starting with a '.' are not listed.
Any ideas?
thanx, Hans-Jörg

**Comment by prasathmani** [comment, marked as answer]
turn on show hidden files in settings
![image](https://user-images.githubusercontent.com/11345598/161429805-108be1b0-6ab4-4abe-8780-eaacc1c484de.png)

**Reply by hjstrauss** [reply]
Hi, I alreqdy did this, but the files are not shown.
Ups, I did it now twice again and out of a sudden it does work now. Thank you very much !
